third_party/move/move-vm/runtime/src/storage/ty_layout_converter.rs

```diff
@@ -307,6 +334,24 @@ pub(crate) trait LayoutConverterBase {
             Type::StructInstantiation { idx, ty_args, .. } => {
                 self.struct_name_to_fully_annotated_layout(*idx, ty_args, count, depth + 1)?
             },
+            Type::Function {
+                args,
+                results,
+                abilities,
+            } => {
+                let mut to_list = |rcs: &[triomphe::Arc<Type>]| {
+                    rcs.iter()
+                        .map(|rc| {
+                            self.type_to_fully_annotated_layout_impl(rc.as_ref(), count, depth + 1)
```

Comment:
Why do we have a count if we never increment it🤦‍♂️ cc @zi0Black @GotenJBZ

Reply:
Lets fix it in some other PR.